src/app/utils/BountyUtils.ts

```diff
@@ -552,7 +560,59 @@ const BountyUtils = {
         });
 
         return customer.lastListMessage;
-    }
+    },
+
+    // bountyCleanUp 
```

Comment:
I really like this - though I wonder if we could make it even more cleanly part of the handler chain (validation --> auth --> activity --> cleanUp)

Reply:
I was going back and forth on that. I didn't want it called after some activities such as List, and for Claim, there are actually 2 bounties that might need work in the case of Evergreen.

I guess in the case of Claim, we can assume that the new bounty is correct since it was just created. When we implement Evergreen on the web side, we'll need to handle that as a special case probably (populating the avatar on the child bounty).

Reply:
Moved it to the handler chain